docs/_docs/dev-guide/eldritch.md

```diff
@@ -253,24 +287,31 @@ mod tests {
 }
 ```
 
-## Example PR
+#### Example PR
```

Comment:
We should remove this example too since it's out of date with the starlark setup.